ops/terraform/services/server/server-regression/iam.tf

```
@@ -326,3 +326,86 @@ resource "aws_iam_role" "glue_trigger" {
     aws_iam_policy.glue.arn
   ]
 }
+
+resource "aws_iam_role" "spice_refresh_role" {
+  name = "bfd-${local.env}-${local.service}-spice-trigger-role"
+  assume_role_policy = jsonencode(
+    {
+      Statement = [
+        {
+          Action = "sts:AssumeRole"
+          Effect = "Allow"
+          Principal = {
+            Service = "lambda.amazonaws.com"
+          }
+        },
+      ]
+      Version = "2012-10-17"
+    }
+  )
+  force_detach_policies = true
+}
+
+resource "aws_iam_policy" "spice_refresh_logs_policy" {
+  name = "bfd-${local.env}-${local.service}-refresh-spice-trigger-logs"
+  description = join("", [
+    "Permissions for the bfd-${local.env}-${local.service}-refresh-spice-trigger Lambda to write to its corresponding ",
```

```suggestion
    "Permissions for the ${local.spice_trigger_lambda_name} Lambda to write to its corresponding ",
```

This manually constructed name could be replaced with the `spice_trigger_lambda_name` local.